apps/extension/inpage.ts

```diff
@@ -1,16 +1,8 @@
-import { WindowPostMessageStream } from '@metamask/post-message-stream';
-
+import { inpageStream } from './src/constants/content-script';
 import { initializeProvider } from './src/dapp-connection-provider';
 
-const CONTENT_SCRIPT = 'oko-contentscript';
-const INPAGE = 'oko-inpage';
```

Comment:
but maybe also be better to replace these too?
<img width="945" alt="image" src="https://user-images.githubusercontent.com/35599200/211281219-5f3fe2e5-8fcd-40e9-af20-8bda3c01a1c7.png">
<img width="952" alt="image" src="https://user-images.githubusercontent.com/35599200/211281292-de5c472d-2fca-4096-a872-2d41d7b7fd53.png">